### DISCUSSION TITLE
Chocolatey packages available

### DISCUSSION THREAD

**Discussion by pdifeo** [amidaware/tacticalrmm#2000] [Q&A] [answered]
I am not a chocolatey user and do not know the level of integration in TacticalRMM.

I have noticed that from ‘software installation’ there are not all the packages available in the repository. For example in the repository there is PDFGear, but from ‘install software’ I cannot find it.

Is there a limitation?

**Comment by wh1te909** [comment, marked as answer]
It's been a few months since we synced with the choco repo so yea some stuff might be missing. I'll try to get it in the next release

**Comment by pdifeo** [comment]
Hi,

there is no news about this. The new version 1.19.4 still does not have some packages such as PDFGear.

**Reply by wh1te909** [reply]
Sorry about that, I just pushed the latest changes. You can import them by running the following commands (as the tactical user) on your trmm server:

```bash
wget -O /rmm/api/tacticalrmm/software/chocos.json 'https://raw.githubusercontent.com/amidaware/tacticalrmm/refs/heads/develop/api/tacticalrmm/software/chocos.json'
cd /rmm/api/tacticalrmm/
source ../env/bin/activate
python manage.py load_chocos
```

After that try loading the list again and you should see PDFGear

**Comment by pdifeo** [comment]
All ok. Thanks

![image](https://github.com/user-attachments/assets/b2c4ed4a-baa5-46bf-81fd-ea440139228e)
